DictDataClasses.py

```diff
@@ -0,0 +1,75 @@
+import random
+import string
+import datetime
+
+class DictDataClasses:
+
+    def __init__(self):
+        self.__n_inst_ref = 14000
+        self.__n_values = 500
+
+    def get_dict(self):
+        return {
+            'inst_id': self.__get_inst_ids(),
+            'asset_class': self.__get_asset_classes(),
+            'COI': self.__get_COIs(),
+            'price': self.__get_prices(),
+            'curr': self.__get_currencies(),
+            'qty': self.__get_quantities(),
+            'type': self.__get_types(),
+            'date': self.__get_dates(),
+            'account': self.__get_accounts(),
+            'inst': self.__get_insts(),
+            'direction': self.__get_directions()
+        }
+
+    def __get_inst_ids(self):
+        possible_chars = string.ascii_uppercase + string.digits
+        return ['ABC' + ''.join([random.choice(possible_chars) for _ in range(5)]) for _ in range(self.__n_inst_ref)]
```

Comment:
Have the value in range be customisable. So have a separate variable to depict its meaning.